Classes/Common/Helper.php

```diff
@@ -258,52 +256,71 @@ public static function getXmlFileAsString($content)
         if (\PHP_VERSION_ID < 80000) {
```

Comment:
Should we remove the code for PHP_VERSION_ID < 80000?

Reply:
Sure, now it is not needed anymore.

Reply:
Done now.